WooCommerce/Classes/ViewRelated/Blaze/BlazeEditAd/BlazeEditAdViewModel.swift

```diff
@@ -180,6 +222,24 @@ extension BlazeEditAdViewModel {
     }
 }
 
+// MARK: AI suggestion
+private extension BlazeEditAdViewModel {
+    func setSelectedSuggestionIfApplicable() {
+        guard let matchingIndex = suggestions.firstIndex(where: { $0.siteName == tagline && $0.textSnippet == description}) else {
+            return selectedSuggestionIndex = nil
+        }
+        selectedSuggestionIndex = matchingIndex
+    }
```

Comment:
nonblocking: because `.firstIndex()` returns nil if it can't find a result, what about simplifying this function to:

```
    func setSelectedSuggestionIfApplicable() {
        selectedSuggestionIndex = suggestions.firstIndex(where: { $0.siteName == tagline && $0.textSnippet == description})
    }
```

feel free to ignore if you intentionally set the nil separately to make the "IfApplicable" part more obvious.

Reply:
Good suggestion. Done in 460d3a2f91666ca894ad16889ddaf509038ca208